### DISCUSSION TITLE
[Bug]  Skywalking UI is giving a blank page when using latest docker image

### DISCUSSION THREAD

**Discussion by Anass-ELGHAOUI** [apache/skywalking#9602] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

UI (apache/skywalking-booster-ui)

### What happened

Skywalking UI is not working when using the lastest docker image, only a blank page is displayed with the following error in the console: `Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'getTimeInfo')`.
![image](https://user-images.githubusercontent.com/32979343/189921221-97bca63c-525b-4dcf-b531-ebd6be461092.png)

### What you expected to happen

functional UI like what was done in the APM distribution

### How to reproduce

run docker-compose with the latest image of Skywalking UI and oap server from apache and use the official docker-compose file https://github.com/apache/skywalking/blob/master/docker/docker-compose.yml to run oap server and the UI.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by guillaume-alvarez** [comment]
Reproduced without docker-compose by running
```
docker run --name oap apache/skywalking-oap-server:9.2.0 -p 12800:12800 -d
docker run --name oap-ui  -p 8080:8080 -e SW_OAP_ADDRESS=http://localhost:12800 apache/skywalking-ui -d
```

**Comment by pg-yang** [comment]
two container  can not  communicate  by  `localhost` . 
Your  docker  config  error

**Comment by kezhenxu94** [comment]
> Reproduced without docker-compose by running
> 
> ```
> docker run --name oap apache/skywalking-oap-server:9.2.0 -p 12800:12800 -d
> docker run --name oap-ui  -p 8080:8080 -e SW_OAP_ADDRESS=http://localhost:12800 apache/skywalking-ui -d
> ```

This is not "reproduce", you didn't use Docker containers correctly at all

**Comment by kezhenxu94** [comment]
@Anass-ELGHAOUI the screenshot you posted basically means your OAP is not reachable, it's either unhealthy or not accessible from the webapp

**Comment by kezhenxu94** [comment]
@Anass-ELGHAOUI please note that https://github.com/apache/skywalking/blob/master/docker/docker-compose.yml is intended to be used after a local build of the Docker images. If you did that, please provide the full steps you run

**Comment by wu-sheng** [comment]
Converting to the discussion as this seems not a bug

**Comment by iTingle** [comment]
@Anass-ELGHAOUI  Hi, the same trouble, do you solve it?

**Reply by iTingle** [reply]
I had solved it.
I use the docker-compose, when dn't set health check well show the trouble。
